src/py/flwr/server/strategy/fedmedian.py

```diff
@@ -118,6 +118,7 @@ def __init__(
             fit_metrics_aggregation_fn=fit_metrics_aggregation_fn,
             evaluate_metrics_aggregation_fn=evaluate_metrics_aggregation_fn,
         )
+        # TODO Aren't these two lines below redundant?
```

Comment:
One function is used to aggregate metrics about the fitting process, the other one is used to aggregate metrics about the evaluation. The metrics I am talking about are the ones returned by those methods [here](https://github.com/adap/flower/blob/main/src/py/flwr/client/numpy_client.py) (`fit()` and `evaluate()`) that are sent back to the server.

Also note that this file (`fedmedian.py`) is not part of this pull request. To avoid conflicts with other PRs I suggest to not modify files that are not meant to be touched by your PRs, otherwise we loose the meaning of opening them for each issue (that is, tracking every change to the main branch) :)
However, if you have any question, a comment in the code is not the right place to ask it. You can use the issue or discussion sections.

Reply:
I see it now, yes it is redundant